messybrainz/db/common.py

```diff
@@ -67,7 +67,7 @@ def create_entity_clusters_for_anomalies(connection,
             logger.info("=" * 80)
             logger.info("Cluster ID: {0}\n".format(cluster_id))
             logger.info("Recordings:")
-            if logger_level >= logging.DEBUG:
+            if logger_level <= logging.DEBUG:
```

Comment:
It looks like this is a difficult condition to understand, especially indicated by the fact that you had to make the change.
I'm looking at the possible values for a logging level: https://docs.python.org/2/library/logging.html#logging-levels
INFO is 20, DEBUG is 10, and NOTSET is 0

If your condition is `if logger_level <= logging.DEBUG:` then you'll never check the condition `if logger_level == logging.INFO:` at https://github.com/metabrainz/messybrainz-server/blob/5bdc4cac50f4851fbea224c321c7a7412bb1e792/messybrainz/db/common.py#L72-L75
What are you trying to do in this condition? I guess you want to show some info when you are running a command, but add more debugging information when logging goes up?